src/com/sap/piper/Notify.groovy

```diff
@@ -0,0 +1,51 @@
+package com.sap.piper
+
+import groovy.text.SimpleTemplateEngine
+
+class Notify implements Serializable {
+    private static enum Severity { ERROR, WARNING }
```

Comment:
Would be cool to have this also for levels `INFO` and `DEBUG`. `DEBUG` of course associated to the `config.debug` flag or somehow similar we have already elsewhere.